crates/extensions/c8y_mapper_ext/src/converter.rs

```diff
@@ -3176,28 +3188,6 @@ pub(crate) mod tests {
         }
     }
 
-    #[tokio::test]
-    async fn try_auto_registration_on_custom_topic() -> Result<(), anyhow::Error> {
```

Comment:
This test isn't valid anymore, since caching of unregistered entity data was implemented.